### PR TITLE
Regenerate with latest gir/gir-files

### PR DESCRIPTION
**Pull request by bilelmoussaoui**

### CODE REVIEW

File: gtk4/src/font_dialog.rs [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+use glib::IsA;
+use std::boxed::Box as Box_;
+use std::pin::Pin;
+use std::ptr;
+
+impl FontDialog {
+    #[doc(alias = "gtk_font_dialog_choose_font_and_features")]
+    #[doc(alias = "gtk_font_dialog_choose_font_and_features_finish")]
+    pub fn choose_font_and_features<
+        P: FnOnce(Result<(pango::FontDescription, glib::GString, pango::Language), glib::Error>)
```
Comment (sdroege) [inline gtk4/src/font_dialog.rs:14]: Is it really guaranteed that either all 3 are NULL and the error is set, or none of them are NULL?

File: gtk4/src/auto/string_list.rs [hunk 2]
```diff
@@ -25,14 +22,6 @@ impl StringList {
         unsafe { from_glib_full(ffi::gtk_string_list_new(strings.to_glib_none().0)) }
     }
 
-    // rustdoc-stripper-ignore-next
-    /// Creates a new builder-pattern struct instance to construct [`StringList`] objects.
-    ///
-    /// This method returns an instance of [`StringListBuilder`](crate::builders::StringListBuilder) which can be used to create [`StringList`] objects.
-    pub fn builder() -> StringListBuilder {
```
Comment (sdroege) [inline gtk4/src/auto/string_list.rs]: Instead you might want to implement the `Extend` and `FromIterator` traits here

Comment (bilelmoussaoui) [reply gtk4/src/auto/string_list.rs]: they are already implemented

Comment (sdroege) [reply gtk4/src/auto/string_list.rs]: Oops

**Comment by sdroege** [comment]
Looks fine to me otherwise :)

File: gtk4/src/font_dialog.rs (hunk 1 above)
Comment (bilelmoussaoui) [reply gtk4/src/font_dialog.rs:14]: Right, FontDescription can be NULL, the two others should always be set if error is NULL.

**Pull request merged**